ocp4/oauth/oauth_test.go

```diff
@@ -20,5 +20,6 @@ func TestTranslateMasterConfig(t *testing.T) {
 	assert.Equal(t, resCrd.Spec.IdentityProviders[0].(identityProviderGitHub).Type, "GitHub")
 	assert.Equal(t, resCrd.Spec.IdentityProviders[1].(identityProviderGitLab).Type, "GitLab")
 	assert.Equal(t, resCrd.Spec.IdentityProviders[2].(identityProviderGoogle).Type, "Google")
-	assert.Equal(t, resCrd.Spec.IdentityProviders[3].(identityProviderHTPasswd).Type, "HTPasswd")
+	assert.Equal(t, resCrd.Spec.IdentityProviders[3].(identityProviderRequestHeader).Type, "RequestHeader")
```

Comment:
Can you please add the provider by alphabetical order, so HTPasswd  would come before RequestHeader?

Reply:
Yes.